usagov/usagov-2021
# USAGOV-1563-Update-Voting-and-elections-All-topics-homepage-icon: Cha…

…nge the voting and elections icon


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1563

## Description

Change the icon of the Voting and elections card in the All topics section.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


1. Go to http://localhost/ 
2. Scroll to the All topics section and check that the Voting and elections card looks like this:
![image](https://github.com/usagov/usagov-2021/assets/39020809/07b1d131-7a51-499b-80c1-0ebf73ac8533)


### Requires New Config

### Requires New Content

### Validation Steps

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
3. Find the commit of this pull request.
4. Build and deploy the changes.
5. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
